src/js/stores/account.js

```diff
@@ -60,8 +60,20 @@ export default Reflux.createStore({
   */
 
   onLogin(username, password, updatePayload) {
+    
     if (localStorage.getItem('webId')){
-      Account.login.completed(localStorage.getItem('webId')) 
+      
+      // Check if the cookie is still valid
+      fetch(`${proxy}/proxy?url=${localStorage.getItem('webId')}`, {
+        method: 'GET',
+        credentials: 'include'
+      }).then((res)=>{
```

Comment:
I tested a bit and it turns out that .then() isn't called if the request failed. If there's a `.catch` then `.catch` is called; if there's only a .then then it throws `Uncaught (in promise) TypeError: Failed to fetch(…)`. I guess we'll have to stick with `.catch`.